src/cli.ts

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env node
+/* eslint no-console:off */
+
+import fs from 'fs/promises';
+import { program } from 'commander';
+import getStdin from 'get-stdin';
+import { jsonToRuntypes } from '.';
+
+const main = async () => {
+  const prog = program
+    .name('json-to-runtypes')
+    .description(
+      'Convert JSON into Runtypes capable of parsing the same JSON shape.',
+    )
+    .option('-i, --input <input file>', 'Path to JSON. Can also be stdin.')
+    .option('-o, --output <output file>', 'Output file.')
```

Comment:
This is optional, right? It'll do stdout if omitted?

Reply:
Both of them are optional, yes :)